main.php

```diff
@@ -14,53 +14,69 @@ function ($errno, $errstr, $errfile, $errline, array $errcontext) {
     $dataFolder = $arguments["data"];
 }
 
+$configFile = $dataFolder . "/config.json";
+if (!file_exists($configFile)) {
+    echo "Config file not found" . "\n";
+    exit(2);
+}
+
 try {
-    $fs = new \Symfony\Component\Filesystem\Filesystem();
+    $jsonDecode = new \Symfony\Component\Serializer\Encoder\JsonDecode(true);
+    $jsonEncode = new \Symfony\Component\Serializer\Encoder\JsonEncode();
+    $config = $jsonDecode->decode(
+        file_get_contents($dataFolder . "/config.json"),
+        \Symfony\Component\Serializer\Encoder\JsonEncoder::FORMAT
+    );
+    $parameters = (new \Symfony\Component\Config\Definition\Processor())->processConfiguration(
```

Comment:
ja nevim no, nebylo by lepsi to rovnou strcit do php-component? uz jsem mel to nutkani minule, ale udrzel jsem se, tedka kdyz se tu pridavaji parametry - to uz mame davno vyreseny

Reply:
no, dobrá!